### ISSUE TITLE
Notification email sent but not received

### ISSUE THREAD

**Issue by adefaria** [JonnyWong16/plexpy#1757]
### Describe the Bug

I set up a newsletter long ago and for months it was working with no issue. Suddenly it stopped setting email. I then set up a Notification Agent to test things. As far as Tautulli is concerned the email has been sent. The log files indicate the email was sent. The problem is the email is never received.

Looking over at my email server I think the problem is that in sending an email using TLS through my email server (I run my own - postfix) the certificate is expired. It is true that on my Synology, the default certificate is expired. Synology said to me that this should not be an issue however I think it is an issue here. Is that the case? If so how to fix it?

Strike that! It turns out my Synology now has a valid cert. Perhaps they updated it with the recent update from DSM 7.0 -> DSM 7.1. So this is perhaps a new cert? Still not sure why it's not being received. Please help me debug this.

### Steps to Reproduce

1. Create a notification agent
2. Test email
3. Check logs - email sent but not received 

### Expected Behavior

Expected the email to be received

### Tautulli Version

 V2.10.1

### Git Branch

master

### Git Commit Hash

a2c589fca46a83b0a244fcc59774ca7c51289b71

### Platform and Version

Synology DSM 7.1

### Python Version

3.9.7

### Browser and Version

Chrome 102.0.5005.61

### Link to Logs

https://gist.github.com/adefaria/93804084f4164c10f81202d1a83897f4#file-tautulli-log

**Comment by JonnyWong16** [comment]
I'm not sure how we can help you. This isn't an issue with Tautulli. Sorry.

**Comment by krowvin** [comment]
Agreed it is almost certainly not an issue with Tautulli. 

But I can offer a few things to try as I stumbled across your question. 

Try using other emailing options within your synology box. Does the web interface for your synology box have the ability to send emails? Do those send OK? 

Try sending emails from your own python script. You mentioned your own email server, It's possible the certificate keystore tautulli uses got out of sync somehow in your synology update with the keystore your server uses? Shot in the dark there.

But running your own script could provide you with more detailed issues to narrow the problem down. 

As a last ditch effort try asking in the [synology](https://www.reddit.com/r/synology/) subreddit to see if similar emailer issues with update  7.1.

**Comment by adefaria** [comment]
Thanks for the prodding. I think I fixed it. Not sure what did it. I found email settings on Synology with which I could test to make sure email from Synology itself can go through my email server and have things delivered. That was tricky due to my own homegrown and restrictive spam filers but I managed to verify that it works. Now Tautulli uses its own email settings and I had been actually CC'ing myself at my Gmail address. When I stopped seeing my CC'ed copy of the newsletter I knew my newsletter email was not going out to others. 

I did some changes WRT to Google. Seems I had removed an SFP TXT record on my domain as I didn't think I needed it anymore and Google was complaining about that. So I readded it. Not sure if this fixed it or not.
